### ISSUE TITLE
Works great but conflicts with standard datepicker

### ISSUE THREAD

**Issue by maiamachine** [spongessuck/gm.datepickerMultiSelect#16]
This is working really well but I'm trying to build something where I have a standard date picker (one day only) as well as a range.  If I put the regular datepicker from angular-ui-bootstrap on the same page as this datepicker, the standard one stops working.  Behavior I'm seeing is a need to double-click in order to get the date buttons to highlight, and the ng-model for the standard date picker stops being updated.  I've tried initiating it with a value, but it gets stuck with that value.  If I initiate it with no value it appears to never pick up any value.  Not sure what's causing this but thought sharing this might help.

**Comment by spongessuck** [comment]
Could you post a sample? The directive shouldn't affect anything not marked 'datepicker'.

**Comment by spongessuck** [comment]
Also see #14.

**Comment by maiamachine** [comment]
I forked a plunkr from your original example and this actually fixed part of my original problem, so my original issue of the single date not registering to the ng-model must be an issue with the implementation in my app.  I will take a look at it.  

I was able to reproduce the issue with #14 however where the dates from the multi-select picker show up as active on the single picker.

Plunkr here: http://plnkr.co/edit/jpyYlN?p=preview
